### ISSUE TITLE
World leak?

### ISSUE THREAD

**Issue by Valleriani** [chylex/Hardcore-Ender-Expansion#140]
HardcoreEnderExpansion  MC-1.7.10  v1.8.3
Java7Checker  MC-1.7.10  v1.1
Forge10.13.4.1448-1.7.10

HardcoreEnderExpansion seems to be slowly leaking. When I remove this mod, it does stop.

I would also like to say I removed chickenchunkloader as well. Though there are some other mods that do give chunk loaders, like railcraft.

[23:38:19] [Server thread/DEBUG] [FML/]: The world 3f2567e9 (world) may have leaked: seen 10 times.
[23:38:49] [Server thread/DEBUG] [FML/]: The world 3f2567e9 (world) may have leaked: seen 15 times.
[23:39:09] [Server thread/DEBUG] [FML/]: The world 3f2567e9 (world) may have leaked: seen 20 times.
[23:39:39] [Server thread/DEBUG] [FML/]: The world 3f2567e9 (world) may have leaked: seen 25 times.

Occurs every 30 seconds, and keeps going higher.

**Comment by chylex** [comment]
https://github.com/chylex/Hardcore-Ender-Expansion/issues/110

**Comment by Valleriani** [comment]
Hello.

I'd like to update this on hoping you have more advice then as I think this involves more then just what was posted there.

I've tested with different mods and these mods also cause the world leak when your mod is loaded with them in my world:

JustAnotherSpawner-0.17.4
GalacticraftCore-1.7-3.0.12.357.jar/MicdoodleCore-1.7-3.0.12.357
Thaumcraft-1.7.10-4.2.3.5
Buildcraft-7.0.22
industrialcraft-2-2.2.770

I've tested each one individually with your mod and it causes it to memory leak. Obviously the Thaumcraft one is the one you heard about. These mods however all play fine with each other if I load them all together and remove your mod. No memory leaks and no issues.

The reason why I was hoping this would be checked more is because I do not think it is their mod causing the issue, I think something is clashing with something on your end.
